packages/devtools/test/inspector_controller_test.dart

```diff
@@ -514,6 +514,14 @@ void main() async {
     });
 
     test('hotReload', () async {
+      if (flutterVersion == '1.2.1') {
```

Comment:
Good idea using the exact flutter version to skip! That makes sure we don't lose the test.